docs/edge_migration.md

```diff
@@ -0,0 +1,73 @@
+# Plans for migrating edge traffic to GCP Ingestion
+
+This document outlines plans to migrate edge traffic to go through the code in this repository.
+
+## Current state
+
+Today, data producers send to a data ingestion stack on AWS as described [here](https://github.com/mozilla/firefox-data-docs/blob/042fddcbf27aa5993ee5578224200a3ef65fd7c7/src/concepts/pipeline/data_pipeline_detail.md#ingestion).
```

Comment:
Nit: while somewhat redundant, leaving the indirect object unspecified makes this statement ambiguous as to what is being sent. I would suggest "data producers send data to a ..." or "data producers produce data to a ...".